oaknational/Oak-Web-Application
# feat: reporting legacy canonical 404s to bugsnag before redirect

## Description

I want a metric showing the proportion of redirects which are returned back to acorn

This change doesn't quite achieve the above, but it forwards 404s from the canonical page to bugsnag ahead of the redirect being returned. If necessary, this data could be used to cross-check the redirects being tracked by the analytics team, but it also gives us a quick and dirty way of viewing how many redirects from OWA to Acorn are occuring.

## How to test

1. Go to https://deploy-preview-2264--oak-web-application.netlify.thenational.academy/pupils/l/video-editing-7a9a/lessons/what-is-video-c4v68d
2. you will be redirected to the Acorn version of the lesson
3. Now go to [bugsnag](https://app.bugsnag.com/oak-national-academy/oak-web-application/errors/?filters[error.status]=open&filters[event.since]=1h&filters[app.release_stage]=preview&filters[event.unhandled]=false&filters[event.severity]=warning&sort=last_seen)
4. You should see the error reported with the title pupils::lesson-overview-legacy-canonical::getStaticProps::pupilLessonOverviewCanonical
